### PR TITLE
[MP-3854] sample app to invite and delete a user from organisation

### PR DESCRIPTION
**Pull request by PawanSran**

### CODE REVIEW

File: Freshservice/sample-custom-actions/actions.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        "user_role": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "user_github_handle",
+        "user_role"
+      ]
+    },
+   "description": "Test description"
```
Comment (jackelstash) [inline Freshservice/sample-custom-actions/actions.json]: ??

File: Freshservice/sample-custom-actions/actions.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ },
+ "deleteUser" : {
+   "display_name" : "test",
```
Comment (jackelstash) [inline Freshservice/sample-custom-actions/actions.json]: ??

File: Freshservice/sample-custom-actions/server/server.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        renderData(null, {'success': true, 'data': response.body});
+      }
+
+      renderData(null,{'success': false, 'error': 'Error while sending request to user'});
+    });
+  },
+
+  deleteUser: function(args) {
+    const options = {
+      url : `https://api.github.com/orgs/${args.iparams.organisation}/memberships/${args.actionparams.user_github_handle}`,
```
Comment (jackelstash) [inline Freshservice/sample-custom-actions/server/server.js]: Isnt `args.actionparams.user_github_handle` supposed to be `args.user_github_handle`?

File: Freshservice/sample-custom-actions/server/server.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+      headers : {
+        Authorization : `Bearer ${args.iparams.gitAdminKey}`,
+        'Content-Type': 'application/json'
+      },
+      method: 'DELETE'
+    };
+
+    request(options, function(error, response) {
+      if(!error && response.statusCode === 204) {
+        renderData(null, {'success': true, 'data': response});
```
Comment (jackelstash) [inline Freshservice/sample-custom-actions/server/server.js]: data => response.body

File: Freshservice/sample-custom-actions/server/server.js (hunk 3 above)
Comment (PawanSran) [reply Freshservice/sample-custom-actions/server/server.js]: No, as part of smi call we get iParams, and actions arguments come as a JSON object actionParams

File: Freshservice/sample-custom-actions/server/server.js (hunk 4 above)
Comment (PawanSran) [reply Freshservice/sample-custom-actions/server/server.js]: the API sends body {} and everything only comes in response object

**Pull request closed without merging**